sourcegraph/cody
# Edit: Show warning on files that are too large

closes https://github.com/sourcegraph/cody/issues/2965

## Description

This PR:
- Adds a warning alongside large files that are @ mentioned in the Edit input
- Shows open tabs by default in the Edit input, as soon as @ is triggered

<img width="673" alt="image" src="https://github.com/sourcegraph/cody/assets/9516420/20810362-4847-481a-9714-2fd40494b85b">

## Test plan

1. Start an edit input, use @ mention
2. Check open tabs are shown immediately
3. Check a warning appears against a large file
